.github/workflows/ci-pipeline.yml

```diff
@@ -49,5 +53,9 @@ jobs:
       run: docker build --target production --tag $DOCKER_USERNAME/todo-app:$GITHUB_SHA --tag $DOCKER_USERNAME/todo-app:prod .
     - name: Push the "production" docker image
       run: docker push $DOCKER_USERNAME/todo-app --all-tags
+    - name: Terraform init
+      run: terraform init
+    - name: Terraform apply
+      run: terraform apply -auto-approve -var 'client_id=${{ secrets.TF_VAR_CLIENT_ID}}' -var 'client_secret=${{ secrets.TF_VAR_CLIENT_SECRET}}' -var 'secret_key=${{ secrets.TF_VAR_SECRET_KEY }}' -var 'env=${{ vars.TF_VAR_ENV }}' -var 'prefix=${{ vars.TF_VAR_PREFIX }}'

```

Comment:
Github auto-capitalises any secrets / vars you add to the repo, and I prefer variable names to be lower case in the terraform. This means they don't get automatically picked up by terraform so I passed them this way 😃